src/main/kotlin/sliv/tool/scene/view/LandmarkStateSynchronizationManager.kt

```diff
@@ -0,0 +1,25 @@
+package sliv.tool.scene.view
+
+import sliv.tool.common.Event
+
+class LandmarkStateSynchronizationManager {
+    val landmarkSelected = Event<LandmarkEventArgs>()
```

Comment:
why not rx or coroutine flow?

Reply:
Replaced event bus here with observable collections, which correspond to the current state, so now there is no events, only observable collections provided by TornadoFX